src/opendr/perception/object_detection_2d/detr/dependencies.ini

```
@@ -0,0 +1,14 @@
+[runtime]
+# 'python' key expects a value using the Python requirements file format
+#  https://pip.pypa.io/en/stable/reference/pip_install/#requirements-file-format 
+python=torch>=1.5.0
+       torchvision>=0.6.0
+       git+https://github.com/cocodataset/cocoapi.git#subdirectory=PythonAPI&egg=pycocotools
+       git+https://github.com/cocodataset/panopticapi.git#egg=panopticapi
+       scipy
+       onnx
+       onnxruntime
+       PIL
```

`PIL` is not a valid module, you should use `pillow` instead (note that version 7.0.0 is the one already used by the toolkit) 
```suggestion
       pillow==7.0.0
```